prototyper/test-kernel/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+target = "riscv64imac-unknown-none-elf"
```

Comment:
This file only works iff we run `cargo` command within the directory containing the `.cargo` path, in this case `prototyper/test-kernel`. Explain why we should have nested cargo config despite we have one in the project root.